### PR TITLE
Update qa-niaid study viewer config

### PR DESCRIPTION
**Pull request by paulineribeyre**
change ES index name so that we get a nicer study viewer URL
use `submitter_id` instead of `project_id` as unique ID. rename it `cmc_unique_id` and make it available in the 3 indices

### CODE REVIEW

File: qa-niaid.planx-pla.net/etlMapping.yaml [hunk 1]
```diff
@@ -17,6 +17,8 @@ mappings:
       - name: description
       - name: title
       - name: project_id
+      - name: cmc_unique_id
+        src: submitter_id
```
Comment (mfshao) [inline qa-niaid.planx-pla.net/etlMapping.yaml:21]: Src is the field name in DD?

Comment (paulineribeyre) [reply qa-niaid.planx-pla.net/etlMapping.yaml:21]: Yes

File: qa-niaid.planx-pla.net/portal/gitops.json [hunk 2]
```diff
@@ -55,10 +55,10 @@
   "studyViewerConfig": [{
     "openMode": "open-first",
     "openFirstRowAccessor": "",
-    "dataType": "cmc",
+    "dataType": "clinical_trials",
     "title": "Studies",
     "titleField": "title",
-    "rowAccessor": "project_id",
+    "rowAccessor": "submitter_id",
```
Comment (mfshao) [inline qa-niaid.planx-pla.net/portal/gitops.json:61]: Should this be cmc_unique_id?

Comment (paulineribeyre) [reply qa-niaid.planx-pla.net/portal/gitops.json:61]: Woops forgot that one

**Review by mfshao — APPROVED** [review]
Lgtm

**Review by atharvar28 — APPROVED** [review]

**Pull request merged**